### PR TITLE
Fix terser version to 4.2.1 in Angular

### LINKED ISSUE #10371: Protractor: e2e tests with Angular are broken
##### **Overview of the issue**

Our current e2e tests with Angular are broken
The tests failed during create/save entities

See https://dev.azure.com/hipster-labs/jhipster-daily-builds/_build

cc @vishal423 

##### **Motivation for or Use Case**

All new PRs will fail, so it needed to be fixed as soon as possible
That's why I'm putting a bounty on this

##### **Browsers and Operating System**

-   [ ] Checking this box is mandatory (this is just to show you read everything)

### PR DESCRIPTION
**Pull request by kaidohallik**
-   Please make sure the below checklist is followed for Pull Requests.

fix #10371

-   [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/templates/angular/package.json.ejs [hunk 1]
```diff
@@ -157,6 +158,11 @@
     "workbox-webpack-plugin": "4.3.1",
     "write-file-webpack-plugin": "4.5.1"
   },
+<%_ if (clientPackageManager === 'yarn') { _%>
```
Comment (vishal423) [inline generators/client/templates/angular/package.json.ejs]: Do you require this as you have included that also as a dependency? Either case, I think we should also add a comment to the corresponding issue that we can track to update/remove this block later on.

Comment (pascalgrimaud) [reply generators/client/templates/angular/package.json.ejs]: Yes, it is needed for Yarn, for a reason I don't remember
No, we can't add comment in JSON file :D

Comment (vishal423) [reply generators/client/templates/angular/package.json.ejs]: we can always add a comment in generator that's used to include this code :smile:

Comment (pascalgrimaud) [reply generators/client/templates/angular/package.json.ejs]: Oh indeed, nice tips !

**Comment by kaidohallik** [comment]
Terser version 4.3.1 is released with fix for that: https://github.com/terser/terser/releases/tag/v4.3.1  
Currently testing, if successful then closing this PR, as transitive dependency then fixes problem.

**Comment by kaidohallik** [comment]
In my testing `terser` version `4.3.1` fixes the problem, so this PR is not needed any more, that's why I'm closing this.

File: generators/client/templates/angular/package.json.ejs (hunk 1 above)
Comment (kaidohallik) [reply generators/client/templates/angular/package.json.ejs]: @vishal423, without this `resolve` part Yarn didn't resolve Terser to version 4.2.1, so different config for Npm and Yarn was needed to work correctly.

**Pull request closed without merging**